src/pal/src/loader/module.cpp

```diff
@@ -1628,6 +1636,12 @@ static HMODULE LOADLoadLibrary(LPCSTR shortAsciiName, BOOL fDynamic)
     LockModuleList();
 
     dl_handle = LOADLoadLibraryDirect(shortAsciiName);
+#ifdef __APPLE__
+    if (!dl_handle && isLibcRequested)
+    {
+        dl_handle = LOADLoadLibraryDirect("/usr/lib/libc.dylib");
```

Comment:
Seconding this comment. If this is to handle varying paths for different versions of OSX there has to be a better way, or some `#include` which can help us resolve this better? Either way I think the semantics and intentions behind the code could be improved. A simple comment might even do?